components/patient-data-sharing/receiver-ui/src/main/resources/data/ReceivePatientConfiguration.xml

```diff
@@ -77,6 +77,13 @@ $xwiki.ssx.use($displayDocumentName)##
     #end
     $doc.display("AllowPushesFromNonListedServers", "edit", $targetObj)
     ##
+  #if ($services.consentauthorizer.consentsGloballyEnabled())
+    {{html clean="false"}}
+    Accept data from old clients which do not ask the user to set consents when pushing: ##
```

Comment:
No good reason, I just wanted to exactly replicate the previous option 1-to-1 (and there checkbox title was within {{html}}), did not check if velocity will generate any extra HTML tags (e.g. an extra <p> or <div> with some style) when {{html}} is included or not.